sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducerClient/CreateBatchOptionsTests.cs

```diff
@@ -55,7 +55,7 @@ public void ToSendOptionsTranslatesProperly()
 
             var sendOptions = options.ToSendOptions();
             Assert.That(sendOptions, Is.Not.Null, "The send options should not be null.");
-            Assert.That(sendOptions, Is.TypeOf<SendOptions>(), "The send options should be a SendOptions instance.");
+            Assert.That(sendOptions, Is.TypeOf<SendEventOptions>(), "The send options should be a SendOptions instance.");
```

Comment:
`should be a SendOptions instance` => `should be a SendEventOptions instance`

Reply:
Going to defer that to the next PR with some misc stuff just to get these due to the large surface.